tv_apps/smart-system/index.html

```diff
@@ -35,6 +35,7 @@
     <script defer src="shared/js/font_size_utils.js"></script>
     <!-- XXX can be removed while gecko support navigator.mozHour12 API -->
     <script defer src="shared/js/date_time_helper.js"></script>
+    <script defer src="tv_shared/js/simple_key_navigation.js"></script>
```

Comment:
This line was placed in <!-- Notifications --> section.
I think simple_key_navigation is not for Notifications only, and it's more like a generic feature in system app. So I move it to <!-- Shared code --> section.